### ISSUE TITLE
update some module not working

### ISSUE THREAD

**Issue by wifi75** [AuxXxilium/arc#696]
Hello 
some update module not working, module 5-6-7-8

![image](https://github.com/AuxXxilium/arc/assets/31816231/c1406612-56eb-42dd-8f36-ae53bb51e945)

![image](https://github.com/AuxXxilium/arc/assets/31816231/d41ce67f-18fb-4c76-b409-b7a089ae6e63)

**Comment by AuxXxilium** [comment]
Yes because github api is blocked in some locations. you have to enter version manual. 

but like i wrote in wiki, you should never update parts to a later version than arc and the best way is to upgrade the loader.
